lib/CtrlNest.pm

```diff
@@ -19,10 +19,7 @@ sub startup ($self) {
   # Connect DB
   my $schema = CtrlNest::Schema->connect($pg_dsn, $pg_user, $pg_pass);
 
-  $self->helper(
-    db    => sub {$schema},
-    const => sub {CtrlNest::Helper::Constants::}
```

Comment:
We should also remove the import for the `use CtrlNest::Helper::Constants` from this file